src/Tools/GetDocumentInsider/src/Commands/GetDocumentCommandWorker.cs

```diff
@@ -371,10 +373,15 @@ private bool GetDocuments(IServiceProvider services)
         return filePath;
     }
 
-    private static string GetDocumentPath(string documentName, string projectName, string outputDirectory)
+    private static string GetDocumentPath(string documentName, string projectName, string outputDirectory, string outputFileName)
     {
         string path;
-        if (string.Equals(DefaultDocumentName, documentName, StringComparison.Ordinal))
+
+        if (!string.IsNullOrWhiteSpace(outputFileName))
```

Comment:
Based on my read of it, I believe this implementation introduces a bug for the scenario where an application has multiple documents configured. In that case, the last document generated will be the one that overrides all the others written to this file path.

I believe the right approach here is to allow customizing the `{ProjectName}` that is used in the string but allow us to contine using document name to distinguish between different documents.

Reply:
Oh ! You're right. I've missed this.
I'll change this for your solution, better than mine.

Thank you !